node-operators/migrate-aut/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Migrate Autonity"
+description: >
+  How to migrate an Autonity node to a new instance in your own environment
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Maintaining node identity
+
+To migrate a node to a new instance the node identity must be preserved. This is done by simply using the existing [P2P node keys: autonityKeys](/concepts/validator/#p2p-node-keys-autonitykeys) and the node's host [ip address](/node-operators/install-aut/#network) for the new node instance.  by doing this the new node instance will have the same node [identifier](/concepts/validator/#validator-identifier) address and [enode url](/glossary/#enode) as the old instance.
```

Comment:
maintaining node identity should only be limited to using same node keys. host ip and port can now be changed by following the migration procedure. however validator must make sure that the instance ip/port is same as regsitered in enode url

Reply:
This page is in the _node operator_ not the _validator_ guide. `updateEnode()` has a constraint check that requires the node is a registered validator.

So you can't change the IP/Port of a plain full node.

I think the answer here is to leave that paragraph as is and to add this note box afterwards:

```
::: {.callout-note title="Additional checks if you are migrating a registered validator node" collapse="false"}
1. Make sure that your validator is not an active member of the consensus committee during the migration. It is recommended that your validator is in a [`paused`](/concepts/validator/#validator-pausing) state before beginning the migration.
2. Verify the node's new IP/Port address is the same as that in your registered enode URL. You can check this by calling `aut validator info --validator` to view the registered enode URL.

If you are operating a validator node it is possible to change your IP/Port address as described in the guide [Migrating validator node to a new IP/Port address](/validators/migrate-vali/#migrating-validator-node-to-a-new-ipport-address).
:::
```

Reply:
Done in commit ed9c3d7d66e1b8ec343e21f854e8c8ed20f19bbc.